core/chainio/avs_writer.go

```diff
@@ -2,15 +2,25 @@ package chainio
 
 import (
 	"context"
+	"fmt"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients/avsregistry"
 	"github.com/Layr-Labs/eigensdk-go/chainio/clients/eth"
 	"github.com/Layr-Labs/eigensdk-go/logging"
 	"github.com/Layr-Labs/eigensdk-go/signer"
+	gethcommon "github.com/ethereum/go-ethereum/common"
 	gethtypes "github.com/ethereum/go-ethereum/core/types"
 	servicemanager "github.com/yetanotherco/aligned_layer/contracts/bindings/AlignedLayerServiceManager"
 	"github.com/yetanotherco/aligned_layer/core/config"
 	"github.com/yetanotherco/aligned_layer/core/utils"
+	"math/big"
+	"time"
+)
+
+const (
+	MaxRetries          = 25
+	SleepTime           = 5 * time.Second
+	IncrementPercentage = 25
```

Comment:
Add something to the name to indicate that is related to Low Fees, since it has too many retries. In particular the sleep time and the increment percentaje

Reply:
Solved in [e4a7da8](https://github.com/yetanotherco/aligned_layer/pull/330/commits/e4a7da8de955f3cc6e358fe981fecdaa233a1861).